### PR TITLE
feat: npm ci with arborist

### PR DESCRIPTION
**Pull request by claudiahdz**
update `npm ci` to use Arborist, lacks proper logging

### CODE REVIEW

File: lib/ci.js [hunk 1]
```diff
@@ -1,47 +1,50 @@
...
+    .then(() => cb())
+    .catch(cb)
+}
 
-  for (const key in npm.config.list[0]) {
-    if (!['log', 'cache'].includes(key)) {
-      opts[key] = npm.config.list[0][key]
+async function ci () {
+  const where = npm.flatOptions.global
+    ? globalTop
```
Comment (isaacs) [inline lib/ci.js]: I think `npm ci` doesn't work for global packages, probably best to just throw an error if global is set, kind of like `audit` does.

**Review by isaacs — COMMENTED** [review]
LGTM!  The logging and auditing can be done once `npm install` and `npm audit` are done.

**Comment by isaacs** [comment]
LGTM!

**Comment by ruyadorno** [comment]
🎉

**Pull request merged**